orion-server/src/main/java/com/pinterest/orion/core/automation/sensor/kafka/KafkaSensor.java

```diff
@@ -65,29 +65,29 @@ private static Map<String, Object> getClusterConfMap(Cluster cluster) {
     return new HashMap<>();
   }
 
-  protected static boolean containsKafkaAdminClientClusterRequestTimeoutMilliseconds(Cluster cluster) {
+  public static boolean containsKafkaAdminClientClusterRequestTimeoutMilliseconds(Cluster cluster) {
```

Comment:
Is this class the right place for these methods? Since they are being needed elsewhere now, should they go to a more common package/class?

Reply:
Agree. I will move them to KafkaCluster class since they are used by Kafka AdminClient. And I wont make them static. The reason of static is the logic was designed for Sensor only